api/openapi/sifchain-openapi.yml

```diff
@@ -1,14 +1,150 @@
 openapi: 3.0.1
 info:
-  title: CLP_GET_REQUESTS
-  description: CLP_GET_REQUESTS
-  version: '0.1'
+  title: Sifnode - OpenAPI spec
+  description: OpenAPI spec for sifnode
+  version: '0.0.1'
 servers:
   - url: 'http://0.0.0.0:1317'
 paths:
+  /clp/getPools:
```

Comment:
@utx0 Do we want the verb names included in the endpoint name? Maybe we should rename this `/pools` ? (and similarly with the other endpoints too).

Reply:
tihs is from cosmos remember.

Reply:
This is how the clp module works in cosmos-sdk land.